### PR TITLE
ID lookup

### PR DESCRIPTION
**Pull request by kwm4385**
Added ID lookup api endpoints for the UI so we don't need ugly urls.

### CODE REVIEW

**Comment by kwm4385** [comment]
Hold on this until I merge in @jhaber 's PR.

**Comment by kwm4385** [comment]
This should be ready to go @tpetr @jhaber

File: BlazarUI/app/scripts/models/BranchDefinition.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+class BranchDefinition extends Model {
+
+  constructor(gitInfo) {
+    super()
+    this.gitInfo = gitInfo;
+  }
+
+  url() {
+    let gitInfo = this.gitInfo;
+    return `${config.apiRoot}/branch/lookup?host=${gitInfo.url}&organization=${gitInfo.org}&repository=${gitInfo.repo}&branch=${gitInfo.branch}`;
```
Comment (tpetr) [inline BlazarUI/app/scripts/models/BranchDefinition.js]: need to keep these URLs consistent -- can you remove the trailing slash before branch?

Comment (kwm4385) [inline BlazarUI/app/scripts/models/BranchDefinition.js]: Shall I remove all instances of this and have the convention be the api root contains the trailing slash?

**Pull request merged**